src/JuMP.jl

```diff
@@ -344,12 +344,29 @@ end
 """
     set_objective_sense(model::Model, sense::MathOptInterface.OptimizationSense)
 
-Sets the objective sense of the model to the given sense.
+Sets the objective sense of the model to the given sense. See
+[`set_objective_function`](@ref) to set the objective function.
```

Comment:
Also mention the `@objective` macro? `set_objective_function` isn't the typical way to set the objective function.